modules/miscutil/lib/upgrades/invenio_2015_01_13_hide_holdings.py

```diff
@@ -0,0 +1,53 @@
+# -*- coding: utf-8 -*-
+#
+# This file is part of Invenio.
+# Copyright (C) 2014 CERN.
+#
+# Invenio is free software; you can redistribute it and/or
+# modify it under the terms of the GNU General Public License as
+# published by the Free Software Foundation; either version 2 of the
+# License, or (at your option) any later version.
+#
+# Invenio is distributed in the hope that it will be useful, but
+# WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+# General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with Invenio; if not, write to the Free Software Foundation,
+# Inc., 59 Temple Place, Suite 330, Boston, MA 02111-1307, USA.
+
+"""Upgrade for explicitly hiding the 'holdings' tab for all collections\
+ except 'Books' without an existing rule"""
+
+from invenio.dbquery import run_sql
+
+depends_on = ['invenio_release_1_1_0']
+
+
+def info():
+    """Upgrade recipe information."""
+    return "Updates the collectiondetailedrecordpagetabs to hide 'holdings' for every collection \
+            except 'Books' without an existing rule."
+
+
+def do_upgrade():
```

Comment:
@jirikuncar Writing upgrade recipe is good in this case, because the removed code would have visible effect even on non-demo installations, which is not wanted.  

@blixhavn However we need to be more careful here.  If people set something for the Home collection, we should reuse the same settings for daugher collections, or leave them untouched.  The current code seems to force Atlantis settings on daugther collections even on those sites where people customised the Home collection settings, without customising daugter collections, which is perhaps not probable but which can occur. When dealing with collection Ci, we should either (1) traverse the collection tree and pick closent parent collection Cj's settings for Ci, or (2) leave the collection Ci settings out, so that the home collection settings would be picked.  (Have not tested, just read the diff.)  And you can end up the upgrade recipe with print warning for people to check out their collection tab settings in the WebSearch Admin interface.